### DISCUSSION TITLE
Welcome to ha-wyzeapi Discussions!

### DISCUSSION THREAD

**Discussion by SecKatie** [SecKatie/ha-wyzeapi#105] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thehyperguy** [comment]
I must be doing something wrong because it doesn't appear that anyone else is having problems with this.  Although I've managed to integrate all of my Wyze devices to HA (sensors, plugs, bulbs), I can't seem to create any automations with any of the Wyze devices.  I can manually execute but that's about it.  The other issue is that it seems to have removed alexa and google assistant integration with the Wyze devices.  Suggestions?

**Reply by SecKatie** [reply]
Please read this section of the readme: https://github.com/JoshuaMulliken/ha-wyzeapi/blob/master/README.md#usage

**Reply by SecKatie** [reply]
It will _**not**_ remove the Alexa and google assistant integrations.

**Comment by aasquith94** [comment]
I feel dumb but i'm new to home assistant and installed HACS just for this. It says to add text to the config file but i can not seem to find it. Is it the home assistant config file? or a HACS config file? or specifically the Wyzeapi integration config file? Thanks

**Reply by LucasPMagno** [reply]
You add the login information to the home assistant config file configuration.yaml. You can edit it by installing Samba from the supervisor addon store or by using a file editor addon also from the supervisor addon store.

**Reply by aasquith94** [reply]
ahh ok well thats why it was so confusing for me. I guess when installing home assistant in a docker it doesnt give me the supervisor. so the question is can I use this without the addon store or should I run a VM?

**Reply by LucasPMagno** [reply]
You will still have a configuration.yaml file that you can edit. However, I would recommend you run it in a VM so you have the supervisor especially if you are now to homeassistant.

**Reply by aasquith94** [reply]
ok interesting. thank you

**Reply by jscherry** [reply]
Does one of the new betas for the thermostat and vacuum include both devices or do they have to be installed separately or either/or? Been using your api for some time and am very happy and appreciate all the work you guys have been putting in on these new products. Just wanted to verify this before installing either the vacuum beta or thermostat beta. Thanks

**Reply by mlhfb** [reply]
I don't think the thermostat is working yet.  I tried installing the Thermostat fork, it's the only Wyze device I have and I get this:  WyzeApi authenticated but could not find any devices.
So probably still a work in progress.

**Comment by aasquith94** [comment]
Ok so I got everything running but I'm having a little trouble with an automation. This was my idea but its not working and I cant seem to figure out why. Any idea? I could also use a single automation that turns off the light after a certain amount of time the sensor is clear. The turn on automation is working fine. the off is not:

automation:
- id: one
  alias: "Turn on light when motion is detected"
  trigger:
    - platform: state
      entity_id: binary_sensor.mud_room_sensor
      to: "true"
  action:
    - service: light.turn_on
      target:
        entity_id: light.mud_room
      data:
        brightness: 255
        kelvin: 418

- id: two
  alias: "Turn off light when motion is no longer detected"
  trigger:
    - platform: state
      entity_id: binary_sensor.mud_room_sensor
      to: "false"
  action:
    - service: light.turn_off
      target:
        entity_id: light.mud_room
      data:
        transition: 1

**Comment by brianhanifin** [comment]
Hello, I am Brian and I have just submitted [my first Pull Request](https://github.com/JoshuaMulliken/ha-wyzeapi/pull/120) to this project! Currently I have the following Wyze products:
1. WyzeCam v2 (monitors our laser cutter, but runs *Dafang Hacks*)
2. Robot Vacuum
3. Outdoor Plug
4. Band

I have been running Home Assistant for 3 years now. Over the years Home Assistant has run on a Raspberry Pi, in Docker, and now on a Virtual Machine on my UNRAID server. I used to be a website developer, but I left that behind when my first son was born 15 years ago. I tinker with devices of all communication types (Z-Wave, Zigbee, Wi-Fi, BLE) and enjoy building my own ESPHome devices on occasion. I occasionally write about my more complex projects on my blog at [brianhanifin.com](https://brianhanifin.com) (for example an Irrigation Controller).

I still love stretching my programming skills to a new language on occasion. Unfortunately I do not have much experience with Python. However I have setup https://insomnia.rest/ on my computer and have been exploring the documented APIs to try to wrap my mind around what this project is working with, to see if I might be able to make little contributions on occasion.

@JoshuaMulliken thank you for starting this!

**Reply by SecKatie** [reply]
Thanks for your PR!!

**Reply by brianhanifin** [reply]
You are very welcome! I am working on setting up the [Development Environment](https://developers.home-assistant.io/docs/development_environment/) to see if I might be able to be more helpful.

**Comment by paulcusick3** [comment]
Is there a way to power off a v2 camera?  I would like to shut off indoor cameras when house occupied.  I can do via IFTTT, but looking to move off IFTTT.

**Reply by paulcusick3** [reply]
bump?

**Reply by paulcusick3** [reply]
nm - i see pull request

**Comment by testventure** [comment]
I'm like 90% sure I must be doing something wrong.  I got HACS installed and Wyze Home Assistant Integration installed but I'm not seeing any of my 5 smart bulbs.  I do see these errors though.   
![2021-04-26 15_49_24-Window](https://user-images.githubusercontent.com/20177937/116160890-adc43b80-a6a7-11eb-8c27-8f1ab32461cb.png)

**Reply by SecKatie** [reply]
Please check that 2fa is disabled and that your password is correct.

**Reply by testventure** [reply]
Ah it was the 2fa.  Thank you.  Your repo here might be why I stick with HA instead of trying to get an expensive hub.

**Comment by LordNex** [comment]
I’m no programmer but I am a network guru. Are you specifying TCP or UDP? If your UDP I’d switch to TCP unless you need access to the broadcast addresses

LoRdNeX

On Apr 27, 2021, at 10:20 AM, testventure ***@***.***> wrote:

﻿

Ah it was the 2fa. Thank you. Your repo here might be why I stick with HA instead of trying to get an expensive hub.

**Comment by LordNex** [comment]
Ohh BTW. I’ve never had 2fa turned on and can verify it’s currently off

Benjamin Powers

On May 4, 2021, at 10:25 AM, paulcusick3 ***@***.***> wrote:

﻿

nm - i see pull request
